### ISSUE TITLE
JIT Crashes on iOS device

### ISSUE THREAD

**Issue by soopsfiya** [hrydgard/ppsspp#11219]
I don't know if this is a duplicate or not, I couldn't find anything relating to this issue from the past 6 months.

### What happens?
Using JIT crashes PPSSPP whenever I launch a game.

### What *should* happen?
Games should start normally.

### What hardware, operating system, and PPSSPP version?  On desktop, GPU matters for graphical issues.
Latest stable PPSSPP version, Latest iOS, iPad mini 4, NOT jailbroken.

**Comment by unknownbrackets** [comment]
I'm not really sure what we can do here.  Any time jit works, it's a bug in iOS.  Maybe Apple fixed that bug.

They don't want PPSSPP's jit to work, which is why it's not really "officially supported".

-[Unknown]

**Comment by vit9696** [comment]
JIT has been working for years just fine on iOS after @hrydgard implemented W^X compatible JIT as I suggested. I have retested it on master on not jailbroken iOS 12.1.4, and believe it is your local setup issue.

Besides, I noticed that some games run into an assertion which seems to be a bug of PPSSPP ARM64 JIT, which I temporarily disabled in https://github.com/hrydgard/ppsspp/pull/11824 to let some games proceed.

Should be closed as invalid.
